.dockerignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
+.git
+.gitignore
+.idea
+log/*
+tmp
+node_modules
+lib
```

Comment:
Should be able to drop `lib` throughout

Reply:
Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/054823863b04f4d041e4dbe7fefd3dd3471d7293